docs/web/using_with_react/ssr/using_with_next.mdx

```diff
@@ -76,6 +76,66 @@ export default MyApp
 The `forceLanguage` prop forces Tolgee to use the language which is provided by Next.js. It will not try to change it
 using its own way.
 
+We can optimize bundle size by only including locale which is needed for rendered page by using getInitialProps.
+
+```jsx
+
+function MyApp({Component, pageProps, locales}: AppProps) {
+    const {locale: activeLocale} = useRouter()
+
+    return <TolgeeProvider
+        ui={UI}
+        forceLanguage={activeLocale}
+        apiKey={process.env.NEXT_PUBLIC_TOLGEE_API_KEY}
+        apiUrl={process.env.NEXT_PUBLIC_TOLGEE_API_URL}
+        staticData={{
+            en: () => import('../i18n/en.json'),
+            cs: () => import('../i18n/cs.json'),
+            ...locales
+        }}
+    >
+        <Component {...pageProps} />
+    </TolgeeProvider>
+}
+
+MyApp.getInitialProps = async (context) => {
+  const locale = context.router.locale;
+  const result = {
+    locales: {
+      [locale]: await import(`../i18n/${locale}.json`),
```

Comment:
What about fallback?

Reply:
I've removed it since in Gatsby's solution you don't have that option and it's questionable if it's necessary, so I removed it also here to stay consistent.

Reply:
It's also removed in example app code